modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/controller/ExportController.scala

```diff
@@ -46,4 +52,62 @@ class ExportController {
       throw new ForbiddenException()
     }
   }
+
+  @PostMapping(value = Array("/tasks/export"), produces = Array(MediaType.APPLICATION_JSON_VALUE))
+  @ResponseBody
+  def exportAllTasksFromList(@PathVariable(value = "cid", required = true) cid: Int, @RequestBody body: JsonNode,
+                             req: HttpServletRequest, res: HttpServletResponse): ResponseEntity[InputStreamResource] = {
+    val user = authService.authorize(req, res)
+    val privilegedByCourse = courseRegistrationService.getParticipants(cid).find(_.user.id == user.id)
+      .exists(p => p.role == CourseRole.DOCENT || p.role == CourseRole.TUTOR)
+
+    if (user.globalRole == GlobalRole.ADMIN || user.globalRole == GlobalRole.MODERATOR || privilegedByCourse) {
+      body.retrive("taskIds").asObject() match {
+        case Some(taskIds) => {
+          if (taskIds.isArray) {
+            logger.info(taskIds.toString)
+            var list: List[Task] = List()
+            taskIds.forEach(id => {
+              val t = taskService.getOne(id.asInt()).get
+              list ::= t
+              logger.info(t.toString)
+            })
+            logger.info(list.toString())
+            val (contentLength, resource) = taskExportService.responseFromTaskId(list, false, -1)
+
+            ResponseEntity.ok()
+              .contentType(MediaType.APPLICATION_OCTET_STREAM)
+              .contentLength(contentLength)
+              .header("Content-Disposition", s"attachment;filename=course_${cid}_only.fbs-export")
+              .body(resource)
+          } else {
+            throw new BadRequestException("No task ids provided")
+          }
+        }
+        case _ => throw new BadRequestException("No tasks selected")
+      }
+    } else {
+      throw new ForbiddenException()
+    }
+  }
+
+  @GetMapping(value = Array("/tasks/export/all"), produces = Array(MediaType.APPLICATION_JSON_VALUE))
```

Comment:
```suggestion
  @GetMapping(value = Array("/tasks/export/all"))
```

This configuration is not necessary.